boards/nxp/imx93_evk/imx93_evk_mimx9352_a55.dts

```diff
@@ -100,17 +99,18 @@
 };
 
 &lpi2c2 {
-	status = "okay";
+	status = "disabled";
 	clock-frequency = <I2C_BITRATE_FAST>;
 	pinctrl-0 = <&i2c2_default>;
 	pinctrl-names = "default";
 
 	mfd0:adp5585@34 {
+		status = "disabled";
```

Comment:
if we move the modifcations ito the imx93_evk_flexcan2.overlay, this should keep as is.